public/doc/swagger-2-v0.1.0.yaml

```diff
@@ -1406,14 +1406,17 @@ definitions:
         example: openshift_default
         type: string
       id:
-        "$ref": "#/definitions/IDReadOnly"
+        type: string
+        pattern: "/^\\d+$/"
```

Comment:
I'd prefer not to duplicate this everywhere, is the issue that `IDReadOnly` depended on `ID`?  (that sounds like a bug in the code generator)
Hopefully this problem is resolved by #140